accounts/accounts_test.go

```diff
@@ -128,7 +128,7 @@ func TestSignRace(t *testing.T) {
 	if err := am.TimedUnlock(a1.Address, "", 15*time.Millisecond); err != nil {
 		t.Fatalf("could not unlock the test account", err)
 	}
-	end := time.Now().Add(80 * time.Millisecond)
+	end := time.Now().Add(500 * time.Millisecond)
 	for time.Now().Before(end) {
```

Comment:
This goroutine is spinning non stop and will not yield the scheduler to the signer. Add `runtime.Gosched()` before repeating the loop. It will fix the failing test if `-cpu=1`